src/runtime_src/core/tools/common/xrt_subcommands.json

```diff
@@ -0,0 +1,79 @@
+{
+    "schema_version" : {
+        "major" : "0x1",
+        "minor" : "0x0"
+     },
+    "xbutil_sub_commands": {
```

Comment:
Encapsulation is your friend here.   It would be better if you had the following:
```
"xbutil" : {
  sub_commands": {
    "kria": {

```

Side Note
Normally, I would have stated that sub_commands should be an array (because it is a collection of commands).  In this case, it is better that it be a dictionary for each sub-command needs to be unique.